Equation.java

```diff
@@ -0,0 +1,68 @@
+import static java.lang.Math.*;
+import java.lang.NumberFormatException;
+/**
+ * solve the quadratic equation
+ * @author Babak Alexey
+ */
+public class Equation {
+
+   /** 
+    * check possibility of the "String -> Double" conversion
+    * @param string  the string to check,
+    * if string has only double variable  - return it
+    */
+   public static double convertString(String string){      
+      try {          
+         return Double.parseDouble(string);
+      } catch (NumberFormatException e) {
+         System.err.println("DataException " + e.getMessage() + " try to input decimal numbers");
+         return 0;     

```

Comment:
Why 0? First of all user can input something like abc which will be parsed as 0 and can masquerade real issue. Secondary this is not java way - you don't know here how exactly this case should be processed - so just throw exception to caller and let it decides how to process it